src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -30,11 +30,11 @@ public JedisPool(final String host) {
     URI uri = URI.create(host);
     if (JedisURIHelper.isValid(uri)) {
       this.internalPool = new GenericObjectPool<Jedis>(new JedisFactory(uri,
-          Protocol.DEFAULT_TIMEOUT, Protocol.DEFAULT_TIMEOUT, null), new GenericObjectPoolConfig());
+          Protocol.DEFAULT_TIMEOUT, Protocol.DEFAULT_TIMEOUT, null), new GenericObjectPoolConfig<Jedis>());
```

Comment:
Do these changes necessary?

Reply:
no, but does resolve a compiler warning about unchecked assignment of a parameterized class

Reply:
though you do bring up a point that there are other constructors that should have this added to them as well to really clean up these compiler warnings, but are unrelated to this specific change